File: courseproject-fitnessapp-asp/Data/Models/FoodDiaryRecord.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace courseproject_fitnessapp_asp.Data.Models
{
    public class FoodDiaryRecord
    {
        [Key]
        public int id { get; set; }
        public int food_id { get; set; }
        public float gramms { get; set; }
        public int meal_id { get; set; }
        public DateTime date_diary { get; set; }
        public int user_id { get; set; }
        public float kcal { get; set; }
        public float protein { get; set; }
        public float fats { get; set; }
        public float carbs { get; set; }
    }
}

File: courseproject-fitnessapp-asp/ClientApp/src/app/models/fooddiaryrecord.ts
export class FoodDiaryRecord {
  constructor(
    public id?: number,
    public date_diary?: Date,
    public food_id?: number,
    public gramms?: number,
    public meal_id?: number,
    public user_id?: number,
    public kcal?: number,
    public protein?: number,
    public fats?: number,
    public carbs?: number,
  ) { }
}
